libminifi/include/core/FlowFile.h

```diff
@@ -310,6 +311,10 @@ struct SpecialFlowAttribute {
   static const std::string ALTERNATE_IDENTIFIER;
   // Flow identifier
   static const std::string FLOW_ID;
+
+  static std::unordered_set<std::string> getSpecialFlowAttributes() {
+    return {PATH, ABSOLUTE_PATH, FILENAME, UUID, priority, MIME_TYPE, DISCARD_REASON, ALTERNATE_IDENTIFIER, FLOW_ID};
```

Comment:
We could use a static array of `const char*` or `std::string_view`, referring to the static strings from above. I want to avoid the overhead of hash tables is possible, but preferably the allocations and the need to store a copy in each processor instance as well.
The static object below is just a flat array of a set of pointers to static storage and sizes, about 144 bytes in size.
```suggestion
  static const auto& getSpecialFlowAttributes() {
    static const std::array<std::string_view, 9> list{
      PATH, ABSOLUTE_PATH, FILENAME, UUID, priority, MIME_TYPE, DISCARD_REASON, ALTERNATE_IDENTIFIER, FLOW_ID
    };
    return list;
```

Reply:
Good idea, updated in e2893bb98eac22b2e9199c5354b33d143494edd5